bundles/org.eclipse.mita.library.stdlib/src/org/eclipse/mita/library/stdlib/RingbufferGenerator.xtend

```diff
@@ -219,7 +219,7 @@ class RingbufferGenerator extends AbstractTypeGenerator {
 					), 
 					true
 				)»
-				«rbRef.code».length++;
+				++«rbRef.code».length;
```

Comment:
is this just a readability issue or did this produce compilation errors?

Reply:
In general, preincrement > postincrement, since the latter generates a temporary copy ;)